.golangci.toml

```diff
@@ -1,2 +1,59 @@
 [run]
 skip-files = ["data/.*\\.gen\\.go", "data/generic_nullable_vector\\.go", "data/generic_vector\\.go"]
+
+[linters-settings.golint]
+min-confidence = 1
+
+[linters-settings.goconst]
+min-len = 2
+min-occurrences = 2
+
+#[linters-settings.gocritic]
+#enabled-tags = [
+  #"diagnostic",
+  #"experimental",
+  #"opinionated",
+  #"performance",
+  #"style",
+#]
+#disabled-checks = [
+  #"dupImport", # https://github.com/go-critic/go-critic/issues/845
+  #"ifElseChain",
+  #"octalLiteral",
+  #"whyNoLint",
+  #"wrapperFunc",
+#]
+
+[linters]
+disable-all = true
+enable = [
+  "bodyclose",
+  "deadcode",
+  "depguard",
+  "dogsled",
+  "errcheck",
+  "gochecknoinits",
+  #"goconst",
```

Comment:
perhaps move the disabled linters out the array and into a commented section about "linters me might enable in the future"

Reply:
Good suggestion @bergquist :+1: